ucsb-cs156-f24/team02-f24-10
# Added Articles Create Page

Closes #58 

How to test: 

<img width="521" alt="image" src="https://github.com/user-attachments/assets/ab896369-aecf-4555-be64-982ca476b7f5">

1. Navigate to the `/articles` page
2. Click on "Create" to navigate to `/articles/create`

<img width="913" alt="image" src="https://github.com/user-attachments/assets/3aabc28c-7278-4e4d-ab35-eb6b613fcb83">
3. Fill out the form, then press submit
<img width="961" alt="image" src="https://github.com/user-attachments/assets/c11c4760-239a-4327-b03e-3ff2a511b4b5">
4. Verify that the data has been added successfully via Swagger under the `/api/articles/all` endpoint

Link to storybook:
https://6722c06126c430949f67f157-wdedzdovxo.chromatic.com/?path=/story/components-articlesform-articlesform--create
